backend/config.default.yaml

```diff
@@ -6,6 +6,10 @@ springServer:
   port: 8086
   shutdownTimeout: 30 #time in second
 
+mdb_service_user:
```

Comment:
not sure how this is being used . some updates in PR description would be great.

Reply:
how does this work for no auth mode?

Reply:
In No Auth mode, MDB ignore this config settings

Reply:
In No Auth mode, MDB ignore this config setting